scripts/machine-launch-script.sh

```diff
@@ -1,57 +1,156 @@
 #!/bin/bash
 
+LAUNCH_STATUS_PATH="/home/centos"
```

Comment:
Can we just drop it in the the working directory  instead of using this absolute path which may not exist on some systems?

Reply:
I was curious where this would go if put in userdata without the full path.  On an Ubuntu 18.04 server, it ended up in /usr/share/doc/cloud-init/userdata.txt

Reply:
On Ubuntu 18.04, it will go into the root of the filesystem `/machine-launchstatus`.  I suspect that it will be the same for anything else using AWS userdata because that all happens using cloud-init.